z3c/dependencychecker/main.py

```diff
@@ -22,6 +22,11 @@ def main():
     report = Report(package_analyzed)
     report.print_report()
 
+    if options.exit_status:
```

Comment:
Both `options.exit_status` and `report.exit_status` have the same name.
It might be clearer to rename the option variable to `exit_with_statuscode`?

But.... It is only used in one place, so leave it be :-)